package.json

```diff
@@ -13,20 +13,22 @@
     "url": "https://www.opensource.org/licenses/mit-license.php"
   },
   "dependencies": {
-    "bitcore-lib-btcz": "git+https://github.com/bitcoinz-wallets/bitcore-lib-btcz.git#v1.0.2",
-    "body-parser": "^1.18.2",
+    "bitcore-lib-btcz": "git+https://github.com/bitcoinz-wallets/bitcore-lib-btcz.git#v2.0.0",
+    "body-parser": "^1.19.0",
     "bookshelf": "^0.12.1",
-    "config": "^1.30.0",
-    "express": "^4.16.3",
-    "google-libphonenumber": "^3.1.3",
-    "joi": "^13.1.2",
-    "knex": "^0.14.4",
+    "config": "^1.31.0",
+    "express": "^4.16.4",
+    "google-libphonenumber": "^3.2.2",
+    "jayson": "^2.0.4",
+    "joi": "^13.7.0",
+    "knex": "^0.14.6",
     "memory-cache": "^0.2.0",
-    "morgan": "^1.9.0",
-    "mysql": "^2.15.0",
-    "nodemon": "^1.17.2",
-    "twilio": "^3.14.0",
-    "zcash": "git+https://github.com/renuzit/zcash.git",
+    "morgan": "^1.9.1",
+    "mysql": "^2.17.1",
+    "nodemon": "^1.19.0",
+    "request": "^2.83.0",
+    "request-promise": "^4.2.2",
```

Comment:
I would probably use `request-promise-native` over `request-promise`. `request-promise` uses the bluebird library which is no longer needed on modern versions of node. `request-promise-native` uses native promises.

Reply:
Updated to request-promise-native 1.0.7